### PR TITLE
Feat: optimize account object with batch and dataloader(issue-1184-account)

### PR DESCRIPTION
**Pull request by Naupio**
## What problem does this PR solve?
link #1184 
optimize account object with batch and dataloader
## Check List
- account object with udt subfield
- account object with brideged_udt subfield
- account object with smart_contract subfield
#### Test
- unit test
#### Task
 - none

### CODE REVIEW

File: lib/godwoken_explorer/graphql/types/account.ex [hunk 1]
```diff
@@ -336,10 +337,11 @@ defmodule GodwokenExplorer.Graphql.Types.Account do
     end
 
     @desc "The mapping smart_contract info of account."
-    field :smart_contract, :smart_contract do
-      description("")
-      resolve(&Resolvers.Account.smart_contract/3)
-    end
+    field :smart_contract, :smart_contract, resolve: dataloader(:graphql)
+
+    # field :smart_contract, :smart_contract do
```
Comment (zmcNotafraid) [inline lib/godwoken_explorer/graphql/types/account.ex:342]: remove these code

**Review by zmcNotafraid — APPROVED** [review]

File: lib/godwoken_explorer/graphql/types/account.ex (hunk 1 above)
Comment (Naupio) [reply lib/godwoken_explorer/graphql/types/account.ex]: fixed

**Comment by Naupio** [comment]
testnet-stg was deployed @FrederLu

**Comment by FrederLu** [comment]
```graphql
query {
	smart_contracts(input:{
		after: "g3QAAAACZAACaWRhO2QABG5hbWVkAANuaWw="
	}) 
	{
		entries {
			id
			name
			account_id
			ckb_balance
			account {
				eth_address
			}
		}
		metadata {
			total_count
			after
			before
		}
	}
}
```
The interface prompts 500.

**Comment by FrederLu** [comment]
> ```graphql
> query {
> 	smart_contracts(input:{
> 		after: "g3QAAAACZAACaWRhO2QABG5hbWVkAANuaWw="
> 	}) 
> 	{
> 		entries {
> 			id
> 			name
> 			account_id
> 			ckb_balance
> 			account {
> 				eth_address
> 			}
> 		}
> 		metadata {
> 			total_count
> 			after
> 			before
> 		}
> 	}
> }
> ```
> 
> The interface prompts 500.

Link:https://github.com/Magickbase/godwoken_explorer/issues/1214#issuecomment-1343807189
verified.

**Review by FrederLu — APPROVED** [review]

**Pull request merged**